### PR TITLE
rules added for OCP-25762

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -91,14 +97,18 @@ check_row_filter_on_page:
 confirm_deletion:
   action: send_delete_string
   action: submit_changes
+check_resource_item:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: This change was already in #560 , temporally add here will rebase after #560 gets merged

**Comment by yapei** [comment]
@SSshahan  Please help review the rules, thanks

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 2]
```diff
@@ -24,6 +24,12 @@ click_create_button:
   - selector:
       xpath: //button[contains(text(),'Create')]
     op: click
+click_confirm_button:
+  element:
+    selector:
+      id: confirm-action
+    type: button
+    op: click
```
Comment (SSshahan) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: can we reuse `submit_changes` rules here?

Comment (yapei) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Yeah, we can. I also would like to change the definition of `submit_changes`

**Comment by yapei** [comment]
rebased the rules  @pruan-rht  Can you help merge?

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml (hunk 1 above)
Comment (yapei) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: already rebased

**Pull request merged**